server/migrations/sql/20191025170040-create-tags-table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE tags(
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL CHECK(name <> ''),
+    type_type_id INT NOT NULL,
```

Comment:
type_type_id => tag_type_id

Reply:
REFERENCES tag_type(id)

Reply:
(oh it on next line, you can do both on same line here